ledger/participant-integration-api/src/main/resources/db/migration/h2database-appendonly/V1__Append_only_schema.sql

```diff
@@ -110,7 +110,7 @@ CREATE TABLE participant_command_completions (
     completion_offset VARCHAR NOT NULL,
     record_time BIGINT NOT NULL,
     application_id VARCHAR NOT NULL,
-    submitters ARRAY NOT NULL,
+    submitters VARCHAR ARRAY NOT NULL,
```

Comment:
can't we just use INTEGER instead?

Reply:
switched it over for everything but event ids, still waiting for CI to tell me how I need to adjust conversions.